midtransclient/subscription.py

```diff
@@ -0,0 +1,128 @@
+from .config import ApiConfig
+from .http_client import HttpClient
+
+class Subscription:
+    """
+    Subscription API is intended for recurring transactions - transactions that deduct customer's funds at a pre-defined time interval.
+    (more params detail refer to: https://api-docs.midtrans.com/#subscription-api)
+    """
+
+    def __init__(self, 
+            is_production=False,
+            server_key='',
+            client_key='',
+            custom_headers=dict(),
+            proxies=dict()):
+
+        self.api_config = ApiConfig(is_production,server_key,client_key,custom_headers,proxies)
+        self.http_client = HttpClient()
+
+    @property
+    def api_config(self):
+        return self.__api_config
+
+    @api_config.setter
+    def api_config(self, new_value):
+        self.__api_config = new_value
+
+    def create(self,parameters=dict()):
+        """
+        Trigger `/v1/subscriptions` API call to Core API
```

Comment:
method name should be consistent with other lib:
- https://github.com/veritrans/veritrans-ruby/blob/master/lib/veritrans/api.rb#L174-L251
- https://github.com/Midtrans/midtrans-go/blob/main/coreapi/clientsubscription.go